# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/resize-observer/change-layout-in-error.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/resize-observer/change-layout-in-error.html.ini
--- a/testing/web-platform/meta/resize-observer/change-layout-in-error.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/resize-observer/change-layout-in-error.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,12 +1,13 @@
 [change-layout-in-error.html]
   [Changing layout in window error handler should not result in lifecyle loop when resize observer loop limit is reached.]
     expected:
-      if not debug and (os == "win") and not webrender and (processor == "x86"): ["PASS", "FAIL"]
-      if not debug and (os == "win") and not webrender and (processor == "x86_64"): ["PASS", "FAIL"]
-      if not debug and (os == "linux") and webrender and not fission: ["PASS", "FAIL"]
-      if debug and (os == "android") and not webrender: ["PASS", "FAIL"]
+      if not debug and (os == "linux") and (processor == "x86_64") and webrender and not fission: ["PASS", "FAIL"]
+      if debug and (os == "win") and webrender and not swgl: ["PASS", "FAIL"]
+      if not debug and (os == "win") and webrender and not fission: ["PASS", "FAIL"]
+      if not debug and (os == "win") and webrender and fission: ["PASS", "FAIL"]
       if not debug and (os == "android") and not webrender: ["PASS", "FAIL"]
       if not debug and (os == "android") and webrender: ["PASS", "FAIL"]
+      if not debug and (os == "linux") and (processor == "x86"): ["PASS", "FAIL"]
       if not debug and (os == "mac"): ["PASS", "FAIL"]
       [FAIL, PASS]
 